database-layout.mermaid

```diff
@@ -1,23 +1,21 @@
-flowchart LR
-    subgraph UserVerification
-        A["User Verification"] --> B["Name"]
-        A --> C["Email"]
-        A --> D["Password"]
-    end
-
-    subgraph Details
-        E["Details"] --> F["Name"]
-        E --> G["Description"]
-        E --> H["Status"]
-        E --> I["File"]
-        E --> J["Created by"]
-        E --> K["Created at"]
-        E --> L["Deleted at"]
-        E --> M["Completed at"]
-        E --> N["Updated at"]
-    end
-
-    subgraph AssignmentTable
-        O["Assignment Table"] --> P["Assigned to"]
-        O --> Q["Assigned at"]
-    end
+%%{init: {'theme': 'base', 'themeVariables': {'primaryColor': '#ffffff', 'edgeLabelBackground': '#ffffff', 'lineColor': '#ffffff'}}}%%
+flowchart TD
+    A["users"] -->A0["id"] 
+    A --> A1["name"]
+    A --> A2["password"]
+    A --> A3["email"]
+    
+    B["tickets"] -->B0["id"]
+    B --> B1["name"]
+    B --> B2["description"]
+    B --> B3["status"]
+    B --> B4["created_by"]
+    B --> B5["created_at"]
+    B --> B6["updated_at"]
+    B --> B7["completed_at"]
+    B --> B8["deleted_at"]
+    
+    C["assignments"] -->C0["id"] 
```

Comment:
I don't think assignments table is required as there is no many-to-many mapping, move assignment columns to ticket table.